crypto/s2n_composite_cipher_aes_sha.c

```diff
@@ -24,14 +24,43 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
 
-int s2n_composite_ciphers_supported()
+/* Silly accessors, but we avoid using version macro guards in multiple places */
+static const EVP_CIPHER *s2n_evp_aes_128_cbc_hmac_sha1(void)
+{
+    /* Symbols for AES-SHA1-CBC composite ciphers were added in Openssl 1.0.1:
+     * See https://www.openssl.org/news/cl101.txt. LibreSSL defines OPENSSL_VERSION_NUMBER to be
+     * 0x20000000L but AES-SHA1-CBC is in LibreSSL since first major release(2.0.0).
+     */
+    #if OPENSSL_VERSION_NUMBER > 0x10000FFFL
```

Comment:
Opted to add a version guard helper instead of just wrapping that value in a constant